hikari/impl/bot.py

```diff
@@ -606,6 +607,16 @@ def unsubscribe(
         # <<inherited docstring from event_dispatcher.EventDispatcher>>
         return self.dispatcher.unsubscribe(event_type, callback)
 
+    def stream(
+        self,
+        event_type: typing.Type[event_dispatcher.EventT_co],
```

Comment:
Can you check if MyPy allows you to pass an abstract event class here for event_type?

We had issues with this elsewhere and had to defer to typing.Any as for some reason MyPy hates you passing abstract types around that get emplaced with generics.

Reply:
should i also change the name for self._ratelimiter then?